### PR TITLE
Add "Apply Asset Tag with InsightVM from Slack"

### PR DESCRIPTION
**Pull request by PhilippBehmer**
WWF Workflow "Apply Asset Tag with InsightVM from Slack"

### CODE REVIEW

File: workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/Apply_Asset_Tag_with_InsightVM_from_Slack.icon [hunk 1]
```diff
@@ -0,0 +1,3381 @@
...
+  "kom": {
+    "komandVersion": "1.60.1-0-g652e5",
+    "komFileVersion": "2.0.0",
+    "exportedAt": "2020-08-21T14:57:25.169400354Z",
+    "workflowVersions": [
+      {
+        "name": "Apply Asset Tag with InsightVM from Slack",
+        "type": "runnable",
+        "version": "",
+        "description": "",
```
Comment (jmcadams-r7) [inline workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/Apply_Asset_Tag_with_InsightVM_from_Slack.icon:11]: Need WF description.

**Comment by jmcadams-r7** [comment]
This is a neat workflow overall...very slick.

**Comment by sengleson-r7** [comment]
@PhilippBehmer this is an awesome workflow!!! Above and beyond what I had in mind, I love how you can create a tag on the fly.
Joey makes a good point in it would be nice to give users a way to cancel the workflow, but I think we could post this as a v1 and improve it.
Can you add a Description in the workflow itself? Once that's in, I think we can merge! Thanks!

**Comment by PhilippBehmer** [comment]
- Updated IVM to 4.8.1
- Updated TC to 1.6.1
- Replaced the "Extract Parameters" step with the new words attribute of the Slack plugin (Good to see my IDEA ticket in action 🎉 )
- Added description

@jmcadams-r7 I wanted to add a cancel button, but ICON only allows 5 buttons. 
If the user wants to cancel he can just do nothing and wait for the timeout (1min)
I did this with "length({{["Join Criticality"].[Responding User]}}) > 0" before, but I just found out that we can add a timeout path to the Slack step and switched to that one.

File: workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+extension: workflow
+products: ["insightconnect"]
+name: Apply_Asset_Tag_with_InsightVM_from_Slack
+title: "Apply Asset Tag with InsightVM from Slack"
+description: This workflow enables you to set asset tags in InsightVM from Slack.
```
Comment (jschipp-r7) [inline workflows/Apply_Asset_Tag_with_InsightVM_from_Slack/workflow.spec.yaml:5]:
```suggestion
description: This workflow enables you to set asset tags in InsightVM from Slack
```

**Review by sengleson-r7 — DISMISSED** [review]

**Comment by PhilippBehmer** [comment]
❯ icon-validate ./Apply_Asset_Tag_with_InsightVM_from_Slack
[*] Validating workflow at ./Apply_Asset_Tag_with_InsightVM_from_Slack

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 27.893ms

**Review by ncramer-r7 — APPROVED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**